sale_order_import/wizard/import_sale_view.xml

```diff
@@ -57,7 +38,7 @@
             </field>
         </record>
 
-        <record id="action_import_sale_order" model="ir.actions.act_window">
+        <record id="import_sale_action" model="ir.actions.act_window">
```

Comment:
The view_id of this record should be "import_sale_view_form" instead of "import_sale_view_form"

Reply:
Fixed. It should be import_sale_view_form so we have fixed it.